db/seeds.d/90-bookmarks.rb

```diff
@@ -0,0 +1,18 @@
+Bookmark.without_auditing do
+  [
+    { :name => 'failed', :query => "status = failed", :controller => 'job_invocations' },
+    { :name => 'succeeded', :query => 'status = succeeded', :controller => 'job_invocations'},
+    { :name => 'running', :query => 'status = running', :controller => 'job_invocations'},
+    { :name => 'pending', :query => 'status = queued or status = running', :controller => 'job_invocations'},
+    { :name => 'recurring', :query => 'recurring = true', :controller=> 'job_invocations'},
```

Comment:
Can we have another bookmark - maybe "recent"? Like `started_at > "24 hours ago"` - only because I don't think it's super intuitive for how that works, so it gives the user something to look at as an example